src/System Application/App/Edit in Excel/src/Filters/EditInExcelFldFilterImpl.Codeunit.al

```diff
@@ -52,6 +70,16 @@ codeunit 1492 "Edit in Excel Fld Filter Impl." implements "Edit in Excel Field F
         FilterCollectionNode.Collection.RemoveAt(Index);
     end;
 
+    procedure GetCollectionType(): Enum "Edit in Excel Filter Collection Type"
+    var
+        TempString: DotNet String;
+    begin
+        TempString := FilterCollectionNode.Operator();
```

Comment:
Is `FilterCollectionNode` guaranteed to be initialized when it gets here?

Reply:
@darjoo to get a reference to a code unit that implements this interface the `"Edit in Excel Filters Impl.":Get()` method is always called which calls `"EditinExcelFldFilterImpl.Initialize()"` that sets the `FilterCollectionNode`. Also similar methods such as `Count()` and `Remove()` would have had the same issue in that code unit.